### PR TITLE
Upgrade/mesos 1.1.0

### PR DESCRIPTION
**Pull request by pfig**
^Review Notes
-----

| Key     | Value                        |
| ------- | ---------------------------- |
| JIRA    | [DD-DD-14683](https://duedil.atlassian.net/browse/DD-DD-14683)  |

### CODE REVIEW

**Review by ekkoshell — APPROVED** [review]
doge_approved

File: src/test/java/com/duedil/mesos/UtilsTest.java [hunk 1]
```diff
@@ -29,14 +30,52 @@ public void testExecutorEndpointThrowsExceptionWithNullInput() {
...
     public void testGetEnvThrowsNpeWhenEnvVarDoesntExist() {
-        String actual = getEnv("MESOS_FROBNITZER");
+        String actual = getEnvOrThrow("MESOS_FROBNITZER");
+    }
+
+    @Test
+    public void testTimeConversionHoldsAllUnits() {
+        TimeConversion tc = TimeConversion.getInstance();
+
+        long oneMili = 1;
```
Comment (ekkoshell) [inline src/test/java/com/duedil/mesos/UtilsTest.java]: constants?

**Review by ekkoshell — APPROVED** [review]
doge_approved

File: src/test/java/com/duedil/mesos/UtilsTest.java (hunk 1 above)
Comment (pfig) [reply src/test/java/com/duedil/mesos/UtilsTest.java]: Yeah, I could make the ones in the class public and use them here. Will do that.

**Review by pfig — COMMENTED** [review]
Agreed

**Pull request merged**